packages/react-client/src/ReactFlightClient.js

```diff
@@ -2423,8 +2436,8 @@ function buildFakeCallStack<T>(
         sourceMap,
         line,
         col,
-        enclosingLine,
-        enclosingCol,
+        useEnclosingLine ? line : enclosingLine,
+        useEnclosingLine ? col : enclosingCol,
```

Comment:
Looking at these two lines in isolation, it seems that the condition should be flipped. Am I missing context, or is this wrong?